modules/get-started/pages/quick-start.adoc

```diff
@@ -36,111 +57,301 @@ docker compose version
-// (step {"action":"runShell", "command": "docker compose up -d --wait", "workingDirectory": "../test-resources/single-broker", "timeout": 50000, "exitCodes": [0,1]})
+.`rpk-profile.yaml`
+[%collapsible]
+====
+[,yaml,subs="attributes+"]
+----
+include::test-resources:attachment$docker-compose/three-brokers/rpk-profile.yaml[]
+----
+====
+
+.`bootstrap.yml`
+[%collapsible]
+====
+[,yaml,subs="attributes+"]
+----
+include::test-resources:attachment$docker-compose/bootstrap.yml[]
+----
+====
+======
+<3> Start Docker Compose.
+
+// (step {"action":"runShell", "command": "docker pull docker.redpanda.com/redpandadata/${REDPANDA_DOCKER_REPO:-redpanda}:${REDPANDA_VERSION:-latest} && docker pull docker.redpanda.com/redpandadata/${CONSOLE_DOCKER_REPO:-console}:${REDPANDA_CONSOLE_VERSION:-latest}", "workingDirectory": "../docker-compose/three-brokers", "timeout": 100000, "exitCodes": [0,1]})
+// (step {"action":"runShell", "command": "docker compose up -d --wait", "workingDirectory": "../docker-compose/three-brokers", "timeout": 50000, "exitCodes": [0,1]})
 // (step {"action":"wait", "duration": 10000})
 
-. Run the following in the directory where you saved the `docker-compose.yml` file:
+When the containers are running, you have:
+
+- A Redpanda cluster with a 30-day free trial license.
+- A running data streaming pipeline generating data into the `logins` and `transactions` topics.
+- A Wasm data transform processing data from `logins` into `edu-filtered-domains`.
+- A secured instance of Redpanda Console with login authentication.
+
+== Explore Redpanda Console
+
+Redpanda Console is a developer-friendly web UI for managing and debugging your Redpanda cluster and your applications. In this section, you'll explore practical examples and scenarios to help you understand how to leverage Redpanda Console for different use cases, including data observability, Redpanda management, access control, and connectivity.
+
+Redpanda Console was deployed as part of the `docker-compose.yml` file and is running locally on port 8080.
+
+=== Log into Redpanda Console
+
+Redpanda Console is configured with plain login credentials (enterprise feature) for an admin user called `jane`. To start using Redpanda Console, you first need to log in with the credentials defined in the `docker-compose.yml` file.
+
+. Open your web browser and go to http://localhost:8080/login.
+. Enter the following credentials:
 +
-```bash
-docker compose up -d
-```
+--
+- Username: `jane`
+- Password: `some-other-secret-password`
+--
 +
-[.no-copy]
+This user is an admin user that has access to all features, including the admin panel.
+
+You should now see an overview of your cluster's status, health, and broker details.
+
```

Comment:
@JakeSCahill you probably explained this, but I can't remember: why do I see it a 1 broker cluster & doc shows 3?

Reply:
@JakeSCahill: I can't explain it, but I started fresh today with updated Docker Desktop & got the three-brokers cluster.